semanticcpg/src/main/scala/io/shiftleft/passes/reachingdef/ReachingDefPass.scala

```diff
@@ -84,7 +84,7 @@ class ReachingDefPass(graph: ScalaGraph) extends CpgPass(graph) {
     method.vertices(Direction.OUT, EdgeTypes.AST).asScala.filter(_.isInstanceOf[nodes.MethodParameterIn]).foreach {
       mPI =>
```

Comment:
mPI is not an understandable abbreviation. Please go at least for: methodParIn or something.